i18n/ro.toml

```diff
@@ -0,0 +1,23 @@
+[category]
+other = "categorie"
+
+[tag]
+other = "etichetă"
+
+[series]
+other = "serie"
+
+[reading_time]
+other = "{{ .Count }} {{ if eq .Count 1 }}minut{{ else if lt .Count 20 }}minute{{ else }}de minute{{ end }} de lectură"
```

Comment:
Have you tried to use `one`, `few`, `many`, and `other` translations here?
It works fine to Slovak. Maybe Romanian is the same case.

https://github.com/luizdepra/hugo-coder/blob/master/i18n/sk.toml#L10-L14

Reply:
I'll need to look into the cut-over for `few` and `many`. It will work for `one`, but the special form is for elements between `[2,19]`.